file-generating/URL-to-PDF api/README.md

```diff
@@ -0,0 +1,166 @@
+# URL-to-PDF API
+
+This is an example that demonstrates packaging a service into a container and running it. The service we will be packaging is an API server that converts a web page into a PDF, i.e., can convert any URL or HTML content to a PDF file or an image (PNG/JPEG) and can be found [here](https://github.com/alvarcarto/url-to-pdf-api). This service is useful when you need to automatically produce PDF files for receipts, weekly reports, invoices, or any other type of content.
+
+Follow the steps below to build the Singularity image required for running this service or you can just download the final image directly from [Container Library](https://cloud.sylabs.io/library/_container/5c3e3b66b0877b0001b0b3fc#container-5c3e3b66b0877b0001b0b3fd), simply run
+`$ singularity pull library://sylabs/doc-examples/url-to-pdf:latest`.
+
+
+### Building the image 
+
+This section will describe the requirements for creating the definition file ([url-to-pdf.def](https://github.com/sushma-98/examples/blob/master/file-generating/URL-to-PDF%20api/url-to-pdf.def)) that will be used to build the container image. `url-to-pdf-api` is based on a Node 8 server that uses a headless version of Chromium called [Puppeteer](https://github.com/GoogleChrome/puppeteer). Let’s first choose a base from which to build our container, in this case the docker image `node:8` which comes pre-installed with Node 8 has been used:
```

Comment:
the ref to this github page should be based on the sylabs/examples repo

Reply:
The whole directory will be merged no? Wen that happens, it will take the link from same repo, the def files are uploaded along with Read Me.

Reply:
I meant this url -> https://github.com/sushma-98/examples/blob/master/file-generating/URL-to-PDF%20api/url-to-pdf.def .. it will be still pointing to your cloned repository on github even though this pr gets merged..